pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -233,9 +252,34 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig, releaseI
 		Registries:            registries,
 		BootImage:             string(*rhcosImage),
 		ClusterDomain:         installConfig.ClusterDomain(),
+		PlatformData:          platformData,
 	}, nil
 }
 
+// getPlatformData returns platform-specific data for bootstrap templates.
+func getPlatformData(config *types.InstallConfig) (platformData platformTemplateData) {
+	switch config.Platform.Name() {
+	case baremetaltypes.Name:
+		_, provNetwork, _ := net.ParseCIDR(config.Platform.BareMetal.ProvisioningNetworkCIDR)
+		cidr, _ := provNetwork.Mask.Size()
+
+		// To4() returns nil when IP is not IPv4
+		ipv6 := provNetwork.IP.To4() == nil
```

Comment:
This bit of code here isn't ideal, we want to derive some information about the ProvisioningNetworkCIDR - IP version, and the CIDR size.  I wonder if I shouldn't implement receiver functions in the BareMetal platform, to return this info.  Or Make ProvisioningNetworkCIDR a type of IPNet.

Calico seems to have implemented the kind of functions we'd need:
  https://github.com/projectcalico/libcalico-go/blob/master/lib/net/ipnet.go